GSoC-Ideas.md

```diff
@@ -1,5 +1,65 @@
 # Ideas for Google Summer of Code projects
 
+## Idea #x: Implementing CHAOSS metrics with Perceval
+
+[ Micro-tasks and place for questions ](#)
+
+### Description
+
+The [GMD Working group](https://github.com/chaoss/wg-gmd) is proposing some metrics
+that are computed with information obtained from software development repositories.
+One of the goals of the working group is to provide reference implementations of
+those metrics, based on the output produced by [Perceval](https://github.com/chaoss/grimoirelab-perceval).
+As an example, still work in progress, check the Python notebook with the
+[reference implementation for commits](https://github.com/chaoss/wg-gmd/blob/master/implementations/Code_Commits.ipynb).
+
+The aims of this idea are as follows:
+
+* Producing Python (Jupyter) notebooks with proposals for reference implementations
+for these metrics. The proposals will follow the usual acceptance mechanism of the working group
+(based on GitHub issues and pull requests). In general, all reference implementations will
+build on the output produced by Perceval for the relevant data source,
+and will explore the peculiarities of the metric and its implementation for that data source.
+
+* Document the notebooks as much as possible, so that any person trying to implement the metric
+can understand not only how to implement it, but also the details that should have into account.
+Documentation should also be suitable for people willing to dig deeper about the details of
+the metric, with the aim of understanding it (not necesarily implementing it).
+
+* When needed, propose changes to Perceval, and maybe other components in GrimoireLab,
+that allow for the implementation of the metric.
+
+* Participate in the acceptance process of the working group.
+
+The aims will require programming in Python, producing Python notebooks, interaction with other
+people in the working group to learn about subtle details of the metrics, and producing
+documentation.
+
+  * _Difficulty:_ Medium
+  * _Requirements:_ Python programming, experience with Python notebooks, skills for producing documentation.
+  * _Recommended:_ Experience in data analytics with Python, if possible involving the use of Pandas, will be a plus. 
+  * _Mentors:_ Jesus M. Gonzalez-Barahona and Valerio Cosentino.
+
+\[Below, template for proposing ideas. Pull request with your idea. \]
+
+### Idea #x:
```

Comment:
Please remove the duplicate idea template (lines 43-60)

Reply:
I did. Thanks for noticing.